platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -419,15 +420,50 @@ object Capture {
          *
          * @param name the name of the operation.
          * @param level the severity of the log.
-         * @fields additional fields to include in the log.
+         * @param fields additional fields to include in the log.
+         * @param startTimeInMs an optional custom start time to use in combination with an `endTimeInMs`
```

Comment:
/nit: is not common convention to include the word "In" (at least in our codebase) we usually just append the unit at the end of the variable, e.g. `startTimeMs` (similar to [c++ conventions](https://accu.org/journals/overload/24/136/mertz_2318/))

Reply:
also it would be good to clarify if this is means unix epoch time or something else